cpu/stm32f3/cpu.c

```diff
@@ -140,5 +140,9 @@ static void cpu_clock_init(void)
     /* disable the HSI if we use the HSE */
     RCC->CR &= ~(RCC_CR_HSION);
     while (RCC->CR & RCC_CR_HSIRDY) {}
+
+    /* swith I2Cx clock source to SYSCLK */
+    RCC->CFGR3 &= ~(RCC_CFGR3_I2CSW);
+    RCC->CFGR3 |= (RCC_CFGR3_I2C1SW_SYSCLK | RCC_CFGR3_I2C2SW_SYSCLK);
```

Comment:
This line should be split and `RCC_CFGR3_I2C2SW_SYSCLK` used only if present (`RCC_CFGR3_I2C3SW_SYSCLK` could also be present):
```c
RCC->CFGR3 |= RCC_CFGR3_I2C1SW_SYSCLK
#ifdef RCC_CFGR3_I2C2SW_SYSCLK
RCC->CFGR3 |= RCC_CFGR3_I2C2SW_SYSCLK
#endif
#ifdef RCC_CFGR3_I2C3SW_SYSCLK
RCC->CFGR3 |= RCC_CFGR3_I2C3SW_SYSCLK
#endif
```
Otherwise it doesn't build, see the CI.